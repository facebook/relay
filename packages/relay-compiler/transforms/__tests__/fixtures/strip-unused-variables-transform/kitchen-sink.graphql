fragment TestFragment on Actor {
  id @skip(if: $usedInDirective)
}

query TestQuery(
  $unusedFirst: Boolean!
  $unusedShared: Boolean!
  $usedInDirective: Boolean!
  $usedInCall: ID!
  $usedInTestQuery2: Boolean!
  $usedInNested: Float!
) {
  node(id: $usedInCall) {
    ...TestFragment
    ...NestedTwoLevels
  }
}

query TestQuery2(
  $unusedSecond: Boolean!
  $unusedShared: Boolean!
  $usedInDirective: Boolean!
  $usedInCall: ID!
  $usedInTestQuery2: Boolean!
) {
  node(id: 4) {
    ... on Actor {
      id @skip(if: $usedInTestQuery2)
    }
  }
}

fragment NestedOneLevel on User {
  ...NestedTwoLevels
}

fragment NestedTwoLevels on User {
  profile_picture(scale: $usedInNested) {
    uri
  }
}
