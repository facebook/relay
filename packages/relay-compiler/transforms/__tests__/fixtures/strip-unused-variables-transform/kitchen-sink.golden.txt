fragment TestFragment on Actor {
  id @skip(if: $usedInDirective)
}

query TestQuery(
  $usedInDirective: Boolean!
  $usedInCall: ID!
) {
  node(id: $usedInCall) {
    ...TestFragment
  }
}

query TestQuery2(
  $usedInTestQuery2: Boolean!
) {
  node(id: 4) {
    ... on Actor {
      id @skip(if: $usedInTestQuery2)
    }
  }
}
